org.eclipse.scanning.api/src/org/eclipse/scanning/api/device/models/IReflectedModel.java

```diff
@@ -1,61 +1,62 @@
 package org.eclipse.scanning.api.device.models;
 
+import static java.util.stream.Collectors.toList;
+
 import java.lang.reflect.Field;
 import java.util.Arrays;
-import java.util.Iterator;
+import java.util.HashSet;
 import java.util.List;
-import java.util.stream.Collectors;
+import java.util.Objects;
+import java.util.Set;
 
 /**
  * In order for models to be automatically added to the mscan
  * command, they must have a way to get from java object to
  * jython command. Any IDetectorModel will have a name and
- * exposure time generated. A refected model can provide a 
+ * exposure time generated. A refected model can provide a
  * method for doing this. This method is defaulted to reflecting
  * the fields out of the model. Please override to provide more
  * fine grained behaviour.
- * 
+ *
  * @author Matthew Gerring
  *
  */
 public interface IReflectedModel {

```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Move constants to a class or enum. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1214)